prql-compiler/src/test.rs

```diff
@@ -2187,6 +2187,32 @@ fn test_switch() {
     "###
     );
 
+    assert_display_snapshot!(compile(
+        r###"
+    from sales
+    select result = match sales.count [
+        eq 1 -> 'one',
```

Comment:
What's the objection to `==1` or just `1`?

Personally I find having operators-as-words bash-y and not so intuitive nor elegant... But open-minded ofc.